src/model/AirLoopHVAC.cpp

```diff
@@ -868,7 +868,7 @@ namespace model {
         airLoopClone.addBranchForHVACComponent(termclone);
       }
 
-      return airLoopClone;
+      return std::move(airLoopClone);
```

Comment:
So this `return std::move` is weird. Normally, you avoid returning a move, because adding move does have a potential effect, though: it prevents the move being elided, because return std::move(foo); is not eligible for NRVO

In this specific case, we return an object of type `AirLoopHVAC` (a child of ModelObject) in a function that returns a ModelObject. On gcc 10 and clang 12, this results in a copy, and this is why it's added here. On gcc 11+ and clang 13+, it is treated as a move already

https://compiler-explorer.com/z/sh676Tfb9